src/benchmarks/micro/corefx/System.Text.Json/Utf8JsonWriter/Perf.Ctor.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Buffers;
+using BenchmarkDotNet.Attributes;
+using MicroBenchmarks;
+
+namespace System.Text.Json
+{
+    [BenchmarkCategory(Categories.CoreFX, Categories.JSON)]
+    public class Perf_Ctor
+    {
+        private ArrayBufferWriter<byte> _arrayBufferWriter;
+        private PooledBufferWriter<byte> _pooledBufferWriter;
+
+        [Params(true, false)]
+        public bool Formatted;
+
+        [Params(true, false)]
+        public bool SkipValidation;
+
+        [Params(true, false)]
+        public bool NewOutput;
```

Comment:
What is the benefit to the NewOutput = true test?  That's more about the perf of two non-public ctors that are standins for user-provided objects.

Reply:
Fair. Will remove.